package.json

```diff
@@ -51,5 +50,14 @@
     "ts-jest": "29.1.3",
     "tslint": "6.1.3",
     "typescript": "5.1.6"
+  },
+  "exports": {
+    "types": "./types/index.d.ts",
+    "worker": {
+      "import": "./lib/index.worker.js"
+    },
+    "default": {
+      "import": "./lib/index.js"
+    }
```

Comment:
I've never used the exports key before. How does this work? Does developers need to import it a certain way to use the worker variant or is it selected automagically?

Reply:
It works automagically, which I was pretty surprised about. It basically does some under the hood runtime detection and picks the entry point that you've defined. I tested this in node 16 & 20, bun, deno and CFWs and it all worked as expected. Pretty neat.

Reply:
@mattgd @PaulAsjes please take a look at [this PR](https://github.com/workos/workos-node/pull/1068), as this commit caused breaking changes